Extensions/audio_plus.css

```diff
@@ -33,6 +33,10 @@
 	border-radius: 6px;
 }
 
+.right_column.has_docked_audio {
```

Comment:
clearer to use the id here - while `#right_column` does also have a `.right_column` class it's confusing to (imo) imply that there could be more than one

plus we're using the id to add/remove the classes so the discrepancy is just kinda weird

Reply:
not a problem ^__^